src/libs/ReportUtils.ts

```diff
@@ -343,6 +330,12 @@ type OptionData = {
     isTaskReport?: boolean | null;
     parentReportAction?: ReportAction;
     displayNamesWithTooltips?: DisplayNameWithTooltips | null;
+    isDefaultRoom?: boolean;
+    isExpenseReport?: boolean;
+    isOptimisticPersonalDetail?: boolean;
+    selected?: boolean;
```

Comment:
Optional recommendation.
```suggestion
    isSelected?: boolean;
```

Reply:
I cannot change it as this is what is used across the app